stdlib/2/wsgiref/__init__.pyi

```diff
@@ -0,0 +1,26 @@
+# Type declaration for a WSGI Function in Python 2
+#
+# This function actually exist, but we need a central place to define the type
+# of a WSGI Application.
+#
+# To correctly use this type stub, utilize the `TYPE_CHECKING` flag in
+# typing:
+#
+# from typing import TYPE_CHECKING
+#
+# if TYPE_CHECKING:
+#   from wsgiref import WSGIFunction
+#
+
+from typing import Callable, Iterable, List, Optional, Tuple, Type, Union
+from types import TracebackType
+
+exc_info = Tuple[Optional[Type[BaseException]],
+                 Optional[BaseException],
+                 Optional[TracebackType]]
+WSGIFunction = Callable[[Dict[Union[unicode, str], Union[unicode, str]],
```

Comment:
Also, this should rather be called WSGICallable as it can very well be a class, as documented in the comment in this example:
https://docs.python.org/3/library/wsgiref.html#examples

Or maybe WSGIApplication which it's called in most frameworks?